web/source/osk/oskManager.ts

```diff
@@ -797,6 +793,8 @@ namespace com.keyman.osk {
         var r=this.getRect();
         this.width=r.width;
         this.height=r.height;
+        this.x = r.left;
+        this.y = r.top;
```

Comment:
This is part of the OSK-repositioning mouse-move handler.  Width and height shouldn't really be changed here; the important parts are the left & top positions.

With these changes, it becomes possible to bypass cookie loads & saves after the initial load.  (See the first commit, which did _exactly_ this until I figured out how to fix the cookies, too.)